src/js/jsx/sections/export/ExportPanel.jsx

```diff
@@ -134,29 +164,37 @@ define(function (require, exports, module) {
             var document = this.props.document,
                 disabled = this.props.disabled,
                 containerContents,
-                addAssetClickHandler;
+                addAssetClickHandler,
+                exportAssetsClickHandler;
 
             if (!document || !this.props.visible || disabled) {
                 containerContents = null;
-            } else if (document.layers.selected.size !== 1) {
+            } else if (document.layers.selected.size > 1) {
+                // FOR NOW - we don't allow multiple selected layers
                 containerContents = (<div>{strings.EXPORT.SELECT_SINGLE_LAYER}</div>);
+                disabled = true;
+            } else if (document.layers.backgroundSelected) {
+                // don't support exports of the background
+                containerContents = null;
+                disabled = true;
+            } else if (document.layers.selected.size === 0 && document.layers.hasArtboard) {
+                // don't support document exports when artboards
+                containerContents = null;
+                disabled = true;
```

Comment:
Correction: I just pushed a commit that removes the restriction.  If the asset generation is too slow or broken we can open bugs against it.  @iwehrman